.ci/e2e/publish_tests/publish_config.json

```diff
@@ -0,0 +1,68 @@
+{
+  "azure_service_principal": {
+    "user_id": "${AZURE_SERVICE_PRINCIPAL_USER}",
```

Comment:
Add helper script to generate spn : https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest